manifests/plugin/write_http.pp

```diff
@@ -1,8 +1,14 @@
-# https://collectd.org/wiki/index.php/Plugin:Write_HTTP
+# @summary Enable write_http plugin
+#
+# @see  https://collectd.org/wiki/index.php/Plugin:Write_HTTP
+#
+# @parameter Manage a collectd-write_http package? If undef a suitable value per OS will be chosen.
+#
 class collectd::plugin::write_http (
   Enum['present', 'absent']          $ensure = 'present',
   Hash[String, Hash[String, Scalar]] $nodes = {},
-  Hash[String, Hash[String, Scalar]] $urls  = {}
+  Hash[String, Hash[String, Scalar]] $urls  = {},
+  Optional[Boolean]                  $manage_package = undef,
```

Comment:
I'm not a big fan of an Optional Boolean. Can't we set the default to true or fals?

Reply:
In this case there are 3 states. 
* Use the default for operating system (so false on debian, true on redhat 8)
* Force to be false.
* Force to be true.